.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,49 @@
+name: Bug Report
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: textarea #description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+
+  - type: textarea #reproduce
+    attributes:
+      label: Environment
+      description:
+        "The OS + version you’re running Kubescape on, e.g Ubuntu 22.04 LTS
+        Version: the version that Kubescape reports when you run `kubescape version`"
```

issue: forms have separate fields for a description and an example (placeholder), but we’re mixing them together as a description

Let’s split them:
```suggestion
      description: "The environment you’re running Kubescape on"
      placeholder: |
        OS: Ubuntu 22.04 LTS
        Version: dev-v2.0.404 (as reported by `kubescape version`)
        Cloud provider: Google Kubernetes Engine
```